tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -58,6 +58,8 @@
     ['window-close-v2', SawyerWindowCloseV2Policy(), .1, 0.92, {}],
     ['button-press-v1', SawyerButtonPressV1Policy(), 0., 0.94, {}],
     ['shelf-place-v2', SawyerShelfPlaceV2Policy(), 0.1, 0.93, {}],
+    ['push-wall-v2', SawyerPushWallV2Policy(), 0.0, 0.8, {}],
```

Comment:
80% success isn't great, especially for a no-noise test. Can this be improved at all?

Reply:
yeah that value was not a tight bound. The value is actually 0.97 with no noise. with noise its ~0.85. Noise affects this one a lot (it drops the puck, stops short of the goal, drifts, etc.)